pyodide_build/pywasmcross.py

```diff
@@ -351,7 +351,11 @@ def calculate_object_exports_readobj(objects: list[str]) -> list[str] | None:
         "-st",
     ] + objects
     completedprocess = subprocess.run(
-        args, encoding="utf8", capture_output=True, env={"PATH": os.environ["PATH"]}
+        args,
+        encoding="utf8",
+        capture_output=True,
+        env={"PATH": os.environ["PATH"]},
+        check=False,
```

Comment:
```suggestion
        args,
        encoding="utf8",
        capture_output=True,
        env={"PATH": os.environ["PATH"]},
        check=True,
```

Reply:
We check the return code of this one manually so `check` should be `False`. Generally if we _don't_ store the return value, then `check` should be `True` and if we do store it it should be `False`.